src/main/java/net/raisetech/raisetechtask9/controller/MovieController.java

```diff
@@ -1,28 +1,29 @@
 package net.raisetech.raisetechtask9.controller;
 
+import net.raisetech.raisetechtask9.ResourceNotFoundException;
 import net.raisetech.raisetechtask9.entity.Movie;
 import net.raisetech.raisetechtask9.service.MovieService;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
-
+import org.springframework.web.bind.annotation.*;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
 public class MovieController {
-
-
-
     private final MovieService movieService;
+
     public MovieController(MovieService movieService) {
         this.movieService = movieService;
     }
-    @GetMapping("/movies")
-    public List<Movie> getMovies(){
-        return movieService.findAll();
-
-        //returnnameService.findAll().stream().map(NameResponse::new).toList();
+    @GetMapping("/movies/{id}")
+    public Optional<Movie> getMovieId(@PathVariable("id") int id) {
```

Comment:
下記のほうがメソッド名としてよいかと思います。
```suggestion
    public Optional<Movie> getMovieById(@PathVariable("id") int id) {
```
または
```suggestion
    public Optional<Movie> getById(@PathVariable("id") int id) {
```